aws/src/main/java/org/apache/iceberg/aws/glue/GlueTableOperations.java

```diff
@@ -66,6 +67,9 @@ class GlueTableOperations extends BaseMetastoreTableOperations {
   // same as org.apache.hadoop.hive.metastore.TableType.EXTERNAL_TABLE
   // more details: https://docs.aws.amazon.com/glue/latest/webapi/API_TableInput.html
   private static final String GLUE_EXTERNAL_TABLE_TYPE = "EXTERNAL_TABLE";
+  private static final String GLUE_ICEBERG_METADATA_REFRESH_MAX_RETRIES =
+          "iceberg.glue.metadata-refresh-max-retries";
```

Comment:
I don' think we should be adding this property for Glue. If we really want to do this, then I think we should parameterize the default max-refresh setting in `BaseMetastoreTableOperations`.

Also having a test similar to https://github.com/apache/iceberg/blob/09b44bb66f4b56bb093a070dec9fcedea98d67ab/hive-metastore/src/test/java/org/apache/iceberg/hive/HiveTableTest.java#L614-L625 would be good

Reply:
Thanks for you comment.  I will parameterize the max-refresh in `BaseMetastoreTableOperations`. Also, I want to confirm, is the default value still the default 20?

Reply:
Sorry, I didn't find a reasonable solution to parameterize in the `BaseMetastoreTableOperations`, I think this may not be very smooth. I would like to ask why we can't configure it in GlueTableOperations.

Reply:
> @nastra Thanks for your comment. I will parameterize the max-refresh in `BaseMetastoreTableOperations`. Also, I want to confirm, is the default value still the default 20?

Reply:
because this seems like something that should be configurable not just for Glue. Why can't we add a new property either to `BaseMetastoreTableOperations` or to `TableProperties`. All places that currently call `void refreshFromMetadataLocation(String newLocation)` would then read that property (or default to 20) and call `void refreshFromMetadataLocation(String newLocation, int numRetries)`

Reply:
yes, I could get you. I also think parameterize max-refresh in the `BaseMetastoreTableOperations` is a good idea. But I mean I can't find a map or conf that I can get the value in the `BaseMetastoreTableOperations` .  If passing parameters, then all of subclasses like `SnowflakeTableOperations` need to adapt it. Also, I think the max-refresh is a catalog level params rather than a table.

Reply:
@nastra hi, I have updated the patch that parameterize max-refresh in the BaseMetastoreTableOperations.  is it make sense? I think this is a little ugly -(.    Looking forward to your reply. thx.

Reply:
I'll take a look this week